lib/dialects/mssql/connection-manager.js

```diff
@@ -1,8 +1,7 @@
 'use strict';
 
 const AbstractConnectionManager = require('../abstract/connection-manager');
-const ResourceLock = require('./resource-lock');
-const Promise = require('../../promise');
+const pLimit = require('p-limit');
```

Comment:
See dicsussion here: https://github.com/sequelize/sequelize/pull/12136
This will lead to conflicts.

Reply:
yeah I had started this branch from the remove using branch, I'll have to rebase it